maptowny-plugin/src/main/java/me/silverwolfg11/maptowny/MapTowny.java

```diff
@@ -122,7 +122,7 @@ private MapPlatform loadPlatform() {
             // If class loaded: "net.pl3x.map.api.Pl3xMapProvider"
             // then load v1, else load v2.
             // No other good differentiating techniques for this exist.
-            String version = classExists("net.pl3x.map.api.Pl3xMapProvider") ? "v1" : "v2";
+            String version = classExists("net.pl3x.map.core.pl3xmap.api()") ? "v2" : "v3";
```

Comment:
.api() is not a class, it's a method. Also you should keep support for v1.

Reply:
The correct class is net.pl3x.map.core.Provider

Reply:
I don't think that part really matters since that part just takes a class that is different from another version to compare them and find out which is the right version, in that case .api() has been added in v3

v1 is almost never used and it really shouldn't even be used anymore, it's outdated and v2-v3 is like x10-20 faster